packages/core/src/global.d.ts

```diff
@@ -11,10 +11,10 @@ declare module 'virtual-routes-ssr' {
 }
 
 declare module 'virtual-site-data' {
-  import { SiteData } from 'shared/types';
```

Comment:
@sanyuan0704 can you help review this change? I'm not sure whether this change matches your expectation

Reply:
The change is useful and solve the type hint problem for siteData.